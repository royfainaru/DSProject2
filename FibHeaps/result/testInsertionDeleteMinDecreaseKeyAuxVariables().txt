╮
├── ╭────────────────╮
│   │ Key: 4         │
│   ├────────────────┤
│   │ Rank: 0        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 2        │
│   │ Prev: (null)   │
│   │ Child: (null)  │
│   ╰────────────────╯
╰── ╭────────────────╮
    │ Key: 2         │
    ├────────────────┤
    │ Rank: 2        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: (null)   │
    │ Prev: 4        │
    │ Child: 5       │
    ╰────────────────╯
    ├── ╭──────────────╮
    │   │ Key: 5       │
    │   ├──────────────┤
    │   │ Rank: 0      │
    │   │ Marked: true │
    │   │ Parent: 2    │
    │   │ Next: 3      │
    │   │ Prev: (null) │
    │   │ Child: (null)│
    │   ╰──────────────╯
    ╰── ╭───────────────╮
        │ Key: 3        │
        ├───────────────┤
        │ Rank: 0       │
        │ Marked: false │
        │ Parent: 2     │
        │ Next: (null)  │
        │ Prev: 5       │
        │ Child: (null) │
        ╰───────────────╯
