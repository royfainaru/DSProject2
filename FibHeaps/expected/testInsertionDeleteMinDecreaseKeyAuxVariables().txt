╮
├── ╭────────────────╮
│   │ Key: 4         │
│   ├────────────────┤
│   │ Rank: 0        │
│   │ Marked: false  │
│   │ Parent: (null) │
│   │ Next: 2        │
│   │ Prev: 2        │
│   │ Child: (null)  │
│   ╰────────────────╯
╰── ╭────────────────╮
    │ Key: 2         │
    ├────────────────┤
    │ Rank: 2        │
    │ Marked: false  │
    │ Parent: (null) │
    │ Next: 4        │
    │ Prev: 4        │
    │ Child: 5       │
    ╰────────────────╯
    ├── ╭──────────────╮
    │   │ Key: 5       │
    │   ├──────────────┤
    │   │ Rank: 0      │
    │   │ Marked: true │
    │   │ Parent: 2    │
    │   │ Next: 3      │
    │   │ Prev: 3      │
    │   │ Child: (null)│
    │   ╰──────────────╯
    ╰── ╭───────────────╮
        │ Key: 3        │
        ├───────────────┤
        │ Rank: 0       │
        │ Marked: false │
        │ Parent: 2     │
        │ Next: 5       │
        │ Prev: 5       │
        │ Child: (null) │
        ╰───────────────╯
